x/btcstaking/types/btc_delegation.go

```diff
@@ -30,7 +30,7 @@ func NewBTCDelegationStatusFromString(statusStr string) (BTCDelegationStatus, er
 	case "any":
 		return BTCDelegationStatus_ANY, nil
 	default:
-		return -1, fmt.Errorf("invalid status string; should be one of {pending, active, unbonding, unbonded, any}")
+		return -1, fmt.Errorf("invalid status string; should be one of {pending, active, unbonded, any}")
```

Comment:
oh can we also handle `verfified` in that function?

Reply:
done at https://github.com/babylonlabs-io/babylon/pull/140/commits/5b9cc82e2b433e6ccf0297bb01ec1613bfe397fc